packages/combobox/src/Combobox.spec.tsx

```diff
@@ -1299,6 +1299,18 @@ describe('packages/combobox', () => {
       expect(onClear).toHaveBeenCalled();
     });
 
+    test('Clear button does not force the menu to reopen', () => {
```

Comment:
@TheSonOfThomp added a test! RFAL

Reply:
Why would clicking the clear button focus the input?

Reply:
I believe that's how native search works, and the recommended w3 Combobox guidelines

Reply:
HTML search clear button focuses the input, but [Combobox guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) don't mention a clear button

https://user-images.githubusercontent.com/2414030/218155592-a7116800-43fc-48aa-9daa-5f8143810dce.mov